src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -497,10 +495,16 @@ Namespace Microsoft.CodeAnalysis.Operations
                     ' Delegate to standard operation handling for this.
                     operandLazy = New Lazy(Of IOperation)(Function() Create(operand))
                 End If
-                Return (operandLazy, IsDelegateCreation:=True)
+                Return (operandLazy, conversion, IsDelegateCreation:=True)
+            End If
+            Dim nestedInfo = GetNestedInfo(operand, conversion.Kind, conversionSyntax, targetType)
+            If nestedInfo.NestedType <> NestedType.None Then
+                Return (CreateNestedDelegateCreationOrExpressionTreeOperand(operand, conversionSyntax),
```

Comment:
>CreateNestedDelegateCreationOrExpressionTreeOperand [](start = 24, length = 51)

Why is it appropriate to use this helper for scenarios that aren't delegate creations or expression tree conversions?